README.md

```diff
@@ -48,3 +48,15 @@ development environment you can use.
    ```bash
    docker-compose run --rm obs-docu daps -vv -d DC-obs-all html
    ```
+
+## Update documentation
+
+The main steps to update the documentation are:
+
+- Fork [obs-docu](https://github.com/openSUSE/obs-docu) project and clone the forked one in your computer.
+- Prepare the development environment following the instructions above.
```

Comment:
I would link to the section so it's ever clearer what has to be done. It can be linked like this: [instructions](https://github.com/openSUSE/obs-docu#development-environment)

Reply:
The instructions are not only in the "Development Environment" section, but in all the previous sections. For this reason I believe it is better to keep it like it is.